pkg/agent/openflow/pipeline.go

```diff
@@ -42,73 +42,73 @@ import (
 
 var (
 	// PipelineClassifierTable is table 0. Packets are forwarded to different pipelines in this table.
-	PipelineClassifierTable = newFeatureTable("PipelineClassifier", 0xff, 0)
+	PipelineClassifierTable = newTable("PipelineClassifier", 0, binding.PipelineAll)
```

Comment:
We do not have a stage name for 0?

Reply:
This table is a special one which shouldn't belong to any stage. Now I place 0 with 255.

Reply:
```
const (
ClassifierStage StageID = iota
	ValidationStage
	ConntrackStateStage
	PreRoutingStage
	EgressSecurityStage
	RoutingStage
	PostRoutingStage
	SwitchingStage
	IngressSecurityStage
	ConntrackStage
	OutputStage

	StageAll StageID = 255


	PipelineARP PipelineID = iota
	PipelineIP
	PipelineMulticast
	PipelineAll PipelineID = 255
)

```
In above code, `StageAll` and `PipelineAll` is for PipelineClassifierTable. Maybe we can still use 255 and define a constant for it.